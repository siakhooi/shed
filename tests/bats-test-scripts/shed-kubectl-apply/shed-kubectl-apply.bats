setup(){
    load '../common-setup'
    common_setup
}
@test "shed-kubectl-apply" {
    run shed-kubectl-apply
    assert_success
    cat_expected "shed-kubectl-apply-usage" | assert_output -
}
@test "shed-kubectl-apply xxx yyy" {
    run shed-kubectl-apply xxx yyy
    assert_success
    cat_expected "shed-kubectl-apply-usage" | assert_output -
}
@test "shed-kubectl-apply xxx" {
    run shed-kubectl-apply xxx
    assert_failure 1
    cat_expected "shed-kubectl-apply-resource-not-exists" | assert_output -
}
@test "shed-kubectl-apply hello-world" {
    run shed-kubectl-apply hello-world
    assert_success
    cat_expected "shed-kubectl-apply" | assert_output -
    shed-kubectl-delete hello-world
    shed-kubectl wait --for delete pod --selector=app=hello-world --timeout 60s
}
