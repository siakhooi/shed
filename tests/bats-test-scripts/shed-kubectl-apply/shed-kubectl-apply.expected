+ shed-kubectl apply -f /root/.shed/kubectl-resources/hello-world
+ kubectl --context kube-dev-1 --namespace ns-application apply -f /root/.shed/kubectl-resources/hello-world
pod/hello-world created
